### PR TITLE
Upgrade Django to 1.8.2 for LTS

### PR DESCRIPTION
**Pull request by y4n9squared**

### CODE REVIEW

**Comment by y4n9squared** [comment]
Looks like pep8 got a few enhancements since the last version we were using. I'll fix the lints in a separate commit.

**Comment by y4n9squared** [comment]
On top of #61

File: web/migrations/0003_auto_20150525_2253.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('web', '0002_load_initial_records'),
+    ]
+
+    operations = [
```
Comment (wangandrewt) [inline web/migrations/0003_auto_20150525_2253.py]: Are these changes because of the upgrade in Django version?

Comment (y4n9squared) [inline web/migrations/0003_auto_20150525_2253.py]: Yeah, there was a breaking change in the base class of the user model.

**Pull request merged**